config/settings.yml

```diff
@@ -97,6 +97,7 @@ feature:
   netids_on: true
   limit_short_description_on: true
   manage_payment_sources_with_users_on: true
+  order_assignment_notifications_on: true
```

Comment:
I went with default `true`; NU and DC will need to have this setting set to `false` on merge.